examples/experimental/trial/imagenet_nas_arch_pytorch/model_def.py

```diff
@@ -55,54 +55,92 @@ def __init__(self, context: PyTorchTrialContext) -> None:
         )
         self.last_epoch_idx = -1
 
-        genotype = Genotype(
-            normal=[
-                ("skip_connect", 1),
-                ("skip_connect", 0),
-                ("sep_conv_3x3", 2),
-                ("sep_conv_3x3", 1),
-                ("sep_conv_5x5", 2),
-                ("sep_conv_3x3", 0),
-                ("sep_conv_5x5", 3),
-                ("sep_conv_5x5", 2),
-            ],
-            normal_concat=range(2, 6),
-            reduce=[
-                ("max_pool_3x3", 1),
-                ("sep_conv_3x3", 0),
-                ("sep_conv_5x5", 1),
-                ("dil_conv_5x5", 2),
-                ("sep_conv_3x3", 1),
-                ("sep_conv_3x3", 3),
-                ("sep_conv_5x5", 1),
-                ("max_pool_3x3", 2),
-            ],
-            reduce_concat=range(2, 6),
+        def build_model():
```

Comment:
nonblocking: I'd move those two functions out of `__init__`, like putting it in a separate function. Otherwise, `__init__` looks very big.